pkg/porter/credentials.go

```diff
@@ -168,13 +171,20 @@ func (p *Porter) GenerateCredentials(opts CredentialOptions) error {
 		fmt.Fprintf(p.Out, "%v", string(data))
 		return nil
 	}
-
+	credentialsDir, err := p.Config.GetCredentialsDir()
+	if err != nil {
+		return errors.Wrap(err, "unable to get credentials directory")
+	}
+	// Make the credentials path if it doesn't exist. MkdirAll does nothing if it already exists
+	// Readable, writable only by the user
+	err = p.Config.FileSystem.MkdirAll(credentialsDir, 0700)
```

Comment:
This isn't handling the error, can you add that quick?

Reply:
yeahhhh i have that in a commit i noticed it wasn't there after i pusheddddd